### PR TITLE
Automatically create a bomberman arena if there are vampires at round-start

### PR DESCRIPTION
**Pull request by DamianX**
:cl:
* rscadd: In the event that roundstart vampires exist, a bomberman arena will be automatically spawned if you want to ghost and play something more fun

### CODE REVIEW

**Comment by I-VAPE-VOX-CLOACA-EVERY-DAY-OF-MY-LIFE** [comment]
A good start, but what it really needs is for someone to replace bomberman with automated thunderdome

**Comment by ShiftyRail** [comment]
Can I call this a meme PR

**Comment by SysadminJeroen** [comment]
>check telescreen
>see bomberman
Huh

Op ma 12 nov. 2018 00:25 schreef ShiftyRail <notifications@github.com:

**Comment by SonixApache** [comment]
why not just make one EVERY round

also arena datums when
instagib arena mode when

File: code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -86,6 +86,8 @@
 	requirements = list(80,60,40,20,20,10,10,10,10,10)
 
 /datum/dynamic_ruleset/roundstart/vampire/execute()
+	new /datum/bomberman_arena(locate(250, 250, 2), pick("15x13 (2 players)","15x15 (4 players)","39x23 (10 players)"))
```
Comment (DeityLink) [inline code/datums/gamemode/dynamic/dynamic_rulesets_roundstart.dm:89]: I kinda wish you placed a landmark in the map or something.

**Comment by Intigracy** [comment]
Just turn it into a landmark and add it to centcomm, there's no reason to make it just for vampire.

**Comment by Exxion** [comment]
I would unironically support this if it were every round and if joining an arena didn't lock you out from being cloned

**Pull request closed without merging**